### ISSUE TITLE
minetest can't responding every 5 sec

### ISSUE THREAD

**Issue by AdityaMH** [luanti-org/luanti#14440]
### Minetest version

```true
Minetest: 5.8.0
```

### Operating system and version

Android 13

### Summary

Minetest can't responding every 5 sec in Android, every loading media then keep pop-up until i'm going to exit app.

Temporary fixing:
deleting cache app. but still annoying need clear cache if want play game.
sometime "can't responding" pop-up didn't happen, but some texture become missing or blank.

100% happen if switch another game.

### Steps to reproduce

Select game until loading screen show "media"

**Comment by grorp** [comment]
Thank you for reporting! This problem has affected many Minetest users on Discord in the past, but you're the first one to actually report it.

Could you please re-test with a recent 5.9.0-dev version, e.g. from https://github.com/minetest/minetest/actions/runs/8179069759 ? There have been some changes to media loading in #14288.

**Comment by AdityaMH** [comment]
i can't install unsigned apk ☹️

**Comment by grorp** [comment]
Here is a signed version:
[app-arm64-v8a-release-signed.apk.zip](https://github.com/minetest/minetest/files/14543178/app-arm64-v8a-release-signed.apk.zip)
[app-armeabi-v7a-release-signed.apk.zip](https://github.com/minetest/minetest/files/14543198/app-armeabi-v7a-release-signed.apk.zip)

It's signed with a different key than the official releases, so you'll probably have to uninstall the old version first :\

**Comment by AdityaMH** [comment]
The test will going long, because i can't back-up games and progress. and redownload all things.

**Comment by AdityaMH** [comment]
now "can't responding" pop-up always show-up every 5 sec when downloading content.

**Comment by AdityaMH** [comment]
Can you separate between dev builds with, actual release? so i can play it both?
i want become tester for dev build, i can't feeling good playing game with lot of bugs.

**Comment by appgurueu** [comment]
I can reproduce "this application does not respond" when starting Mineclone2 on a sufficiently slow debug build with asan enabled on Ubuntu. I imagine something similar happens on slower phones. The thread that is supposed to tell the OS that Minetest is still alive is probably being starved, or is busy doing other things.

**Comment by rubenwardy** [comment]
> The thread that is supposed to tell the OS that Minetest is still alive is probably being starved, or is busy doing other things.

Yeah, this thread is the main thread. The way a program tells the OS that it is still alive is by processing events. So if the main thread is blocked by creating a server, it won't be processing events

**Comment by nininik0** [comment]
yeah..... I have this issue as well. considering the fact that it sometimes does the no response message in media or creating server and shutting down.

**Comment by sfan5** [comment]
Needs re-testing.
